src/env/convert_array_to_env.c

```diff
@@ -0,0 +1,32 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   convert_array_to_env.c                             :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: yliu <yliu@student.42.jp>                  +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/09/10 12:35:42 by yliu              #+#    #+#             */
+/*   Updated: 2024/09/10 14:39:14 by yliu             ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "env.h"
+
+t_env_list	*convert_array_to_env(char **envp)
+{
+	t_env_list	*env_list;
+	char		*equal_index;
```

Comment:
`equal_index` sounds like integer value.
How about we call it `equal_ptr` instead?

Reply:
37deabe strdup -> xstrdup